modules/rosa-prereq-roles-overview.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// * rosa_planning/rosa-hcp-prepare-iam-resources.adoc
+
+:_mod-docs-content-type: MODULE
+[id="rosa-prereq-roles-overview"]
+= Overview of required roles
+
+To create and manage your {product-title} cluster, you must create several account-wide and cluster-wide roles. If you intend to use {cluster-manager} to create or manage your cluster, some additional roles are required.
+
+To create and manage clusters:: Several account-wide roles are required to create and manage ROSA clusters. These roles only need to be created once per AWS account, and do not need to be created fresh for each cluster. You can specify your own prefix, or use the default prefix (`ManagedOpenShift`).
++
+The required account-wide roles are:
```

Comment:
```suggestion
See the following required account-wide roles:
```

Reply:
Went with "The following account-wide roles are required:"